### PR TITLE
feat: Added bulgarian (bg) translations

### PR DESCRIPTION
**Pull request by null-none**
### Checklist:

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you linted your code locally prior to submission?
* [x] Have you successfully ran app with your changes locally?

### CODE REVIEW

File: src/shared/locales/all.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import eleLocaleJa from 'element-ui/lib/locale/lang/ja'
 import eleLocaleKo from 'element-ui/lib/locale/lang/ko'
 import eleLocalePtBR from 'element-ui/lib/locale/lang/pt-br'
 import eleLocaleRu from 'element-ui/lib/locale/lang/ru-RU'
+import eleLocaleBg from 'element-ui/lib/locale/lang/bg'
```
Comment (agalwood) [inline src/shared/locales/all.js:12]: Please sort the locale key in alphabetical order.

```javascript
import eleLocaleBg from 'element-ui/lib/locale/lang/bg'
import eleLocaleCa from 'element-ui/lib/locale/lang/ca'
import eleLocaleDe from 'element-ui/lib/locale/lang/de'
...
```

File: src/shared/locales/all.js [hunk 2]
```diff
@@ -20,6 +21,7 @@ import appLocaleEnUS from '@shared/locales/en-US'
 import appLocaleEs from '@shared/locales/es'
 import appLocaleFa from '@shared/locales/fa'
 import appLocaleFr from '@shared/locales/fr'
+import appLocaleBg from '@shared/locales/bg'
```
Comment (agalwood) [inline src/shared/locales/all.js:24]: Please sort the locale key in alphabetical order.

```javascript
...
import appLocaleBg from '@shared/locales/bg'
import appLocaleCa from '@shared/locales/ca'
import appLocaleDe from '@shared/locales/de'
...
```

**Comment by agalwood** [comment]
Thank you for your translation : )

**Comment by null-none** [comment]
@agalwood Fixed bugs

**Comment by agalwood** [comment]
Sorry, I found one less during the review.

> src/shared/locales/index.js

**Comment by null-none** [comment]
@agalwood 

lines 73-76

```js
  {
    value: 'bg',
    label: 'Българският език'
  }
```

**Comment by agalwood** [comment]
src/shared/locales/index.js

```javascript
...
{
  value: 'bg',
  label: 'Българският език'
},
{
  value: 'ca',
  label: 'Català'
},
...
```

I found a missing translation:

https://github.com/agalwood/Motrix/blob/0223e691ff49deb6d183b40ef2edbad03322a119/src/shared/locales/en-US/preferences.js#L23

**Comment by null-none** [comment]
@agalwood check please

File: src/shared/locales/index.js [hunk 3]
```diff
@@ -70,10 +74,6 @@ export const availableLanguages = [
     value: 'uk',
     label: 'Українська'
   },
```
Comment (agalwood) [inline src/shared/locales/index.js:76]: ✘  http://eslint.org/docs/rules/comma-dangle  Unexpected trailing comma
    src/shared/locales/index.js:76:4

**Pull request merged**